model/insights.go

```diff
@@ -50,6 +50,32 @@ type TopChannel struct {
 	MessageCount int64       `json:"message_count"`
 }
 
+// Top Threads
+type TopThreadList struct {
+	InsightsListData
+	Items []*TopThread `json:"items"`
+}
+
+type TopThread struct {
+	PostId          string                      `json:"post_id"`
+	ReplyCount      int64                       `json:"reply_count"`
+	ChannelId       string                      `json:"channel_id"`
+	DisplayName     string                      `json:"channel_display_name"`
+	Name            string                      `json:"channel_name"`
+	Message         string                      `json:"message"`
+	Participants    StringArray                 `json:"participants"`
+	UserId          string                      `json:"user_id"`
```

Comment:
As discussed,
1) we have to separate the DB output and Response model if we mind the extra `UserId` here. Similar to PostId, this helps to populate UserInformation later, just like PostId populates `Post`.
2) We can use `Query` instead of `Select` but that's sparsely used in the repository.